src/server/components/lists/searches/translators-interpreters.ts

```diff
@@ -29,6 +29,26 @@ export const translatorsInterpretersQuestionsSequence = [
   QuestionName.readDisclaimer,
 ];
 
+type TranslatorInterpreterServicesProvided = "translation" | "interpretation" | "all";
+
+function makeResultsTitle(country: string | undefined, servicesProvided: string[] | TranslatorInterpreterServicesProvided[]): string {
```

Comment:
I don't get why `servicesProvided` would have type array of strings or an array of set string values

I feel like this `servicesProvided: string[]` negates `TranslatorInterpreterServicesProvided[]`. 

Would it make sense to remove `string[]`?

Reply:
oops,
1. "all" shouldn't be part of this - I misread the query params
  `&servicesProvided=translation,interpretation&interpreterServices=All` <- "all" shouldn't appear in the array at all.
2. country should just be string
3. yeah you're right, I do actually end up casting the values so no need for string[]